nemo/utils/exp_manager.py

```diff
@@ -65,10 +65,10 @@ def exp_manager(trainer: 'pytorch_lightning.Trainer', cfg: Optional[Union[DictCo
             - files_to_copy (list): A list of files to copy to the experiment logging directory. Defaults to None which
                 copies no files.
     """
-    schema = OmegaConf.structured(ExpManagerConfig)
     if cfg is None:
-        cfg = schema
-    elif isinstance(cfg, dict):
```

Comment:
We should cast dicts to DictConfig and support this use case for user ease of use, maybe